front-end/components/SettingsPanel/panels/Personalize/personalize.vue

```diff
@@ -5,11 +5,67 @@
             To increase the size of the dashboard, please use the built in zoom functionality using `ctrl +` and `ctrl
             -`.
         </p>
+        <p><strong>Hide cursor:</strong></p>
+        <button class="option" @click="disableCursorHide" :class="{ current: !cursorHidden }">Don't hide cursor</button>
+        <button class="option" @click="enableCursorHide" :class="{ current: cursorHidden }">Hide cursor after:</button>
+        <div id="cursorHiddenTimeout" v-if="cursorHidden">
+            <input v-model.number="cursorHiddenTimeout" type="number" class="text" /> Miliseconds
+        </div>
     </div>
 </template>
 
 <script>
-export default {};
+import CIMonitorLogo from '../../../EmptyBoard/logo.png';
+import {
+    SETTINGS_SET_CURSORHIDDEN_ON,
+    SETTINGS_SET_CURSORHIDDEN_OFF,
+    SETTINGS_SET_CURSORHIDDEN_TIMEOUT,
+} from '../../../../store/StaticMutations';
+
+export default {
+    methods: {
+        enableCursorHide() {
+            Notification.requestPermission().then(() => {
```

Comment:
Personally I do like the notifications, but looking at the UX it's unexpected behavior. You will only get the "do you want to enable notifications for ..." popup once. And since this unexpected you might miss out on the chance to properly activate the notifications. So I would rather see the notification removed here. Do you agree?

Reply:
Actually, I haven't been able to get the notifications to show to be honest, so I'll happily remove them.